### DISCUSSION TITLE
Can the Home icon be removed?

### DISCUSSION THREAD

**Discussion by darashayda** [requarks/wiki#5992] [Help / Questions]
Please see this

http://wiki.ccnstudios.com/en/home/freeform/intro  

Works! 

Click on my lowercase `home` and again works as expected. 

Now click on the Home icon and we are redirected to admin area. 

It is fine but will confuse our mostly retail future users. 

Is there anyway to remove or hide the Home icon?

Thanx

Dara

**Comment by NGPixel** [comment]
Why are you using a `/home` subfolder instead of just `/freeform/intro`?

**Reply by darashayda** [reply]
because we need multiple folders under a home directory for multiple parts of our cloud systems. 

We finally created what we needed per the GENERAL page of WIKI admin:

http://wiki.ccnstudios.com
 
Underneath this then we can hang pages 

http://wiki.ccnstudios.com/cloudfunc/geometry/emboss

We tried many other ways and were not able to get this link supported. Nor any pages at admin pointed to the proper way of setting up a link like above. 

In conclusion we need 

http://wiki.ccnstudios.com/category/../.../.../page

**Reply by darashayda** [reply]
All is fine and makes sense with this wiki except the home setup

**Comment by darashayda** [comment]
Hello again 

Please see:
http://wiki.ccnstudios.com/freeform/geometry/3DFractals/examples  

Please make suggestions as  how to present these in WIKI  as both scientific computing pages + Arts items . 

In other words any ideas on the formatting and possibly other WIKI tools you might suggest. 

Mind you , you have no obligation to respond. 


Dara
